new_src/node.py

```
@@ -46,7 +51,9 @@ def _inject_uploaded_context_if_any(state: State, msgs: list[AnyMessage]) -> lis
             lines.append(f"- {snippet}\n  [◆ 업로드 파일] {src}")
         context_block = "아래는 사용자가 업로드한 파일에서 검색된 관련 구문입니다. 가능한 한 이를 우선 참고해 답변하세요:\n" + "\n".join(lines)
         msgs = msgs + [SystemMessage(content=context_block)]
-    except Exception:
+    except Exception as e:
```

Ruff (0.14.3)

Do not catch blind exception: `Exception`

(BLE001)